test/spec/modules/unrulyBidAdapter_spec.js

```diff
@@ -42,9 +42,40 @@ describe('UnrulyAdapter', function () {
     }
   }
 
-  const createExchangeResponse = (...bids) => ({
-    body: {bids}
-  });
+  function createOutStreamExchangeAuctionConfig() {
+    return {
+      'seller': 'https://targeting.unrulymedia.com',
+      'decisionLogicURL': 'https://targeting.unrulymedia.com/padecisionlogic',
```

Comment:
this doesnt exists

Reply:
The URL's updated and live. Note that this was just for testing purposes and not used in the production adapter.